src/lib/containers/StackedRowHomebrew.tsx

```diff
@@ -141,16 +141,17 @@ export default class StackedRowHomebrew extends
 
   public getHoverText(xData: any) {
     const { index, hoverText } = this.state
-    const hoverTextDisplay = index === -1 ? xData[0] && xData[0].value : hoverText
+    const hoverTextDisplay = index === -1 ? (xData[0] && xData[0].value) : hoverText
     return (
-      <React.Fragment>
-        <span className="SRC-text-cap">
+      <span id="barChartHoverText">
```

Comment:
Danger!  The [spec](https://www.w3.org/TR/html401/struct/global.html#h-7.5.2) indicates that the id must be unique across the doc.  If someone uses more than one of these in the same page, they'll have an issue.  If this is for css selectors, could you define a standard where portal devs need to add all SRC components inside an element of known ID (like id="syn-react-components")?  For our portals, this could be the body's id!

Reply:
Ah right, since all default create-react-apps have their body tag as "root" I can target that, I will make sure to document this. Are you okay with this?

Reply:
Sounds reasonable to me!  And if we get to a place where this project build can be exported for use in vanilla js apps in the browser (integration ready for Agora and SWC) then we'll just need to remember to designate an element with id="root"!